### DISCUSSION TITLE
Custom colors in visualization

### DISCUSSION THREAD

**Discussion by andrew-j-levy** [google/brax#210] [General]
Hello, 

Is there a simple way to customize the color of bodies in the visualization instead of using the default beige color?  For instance, is there a change that can be made to the config file to set the color of a body?

Thanks!

**Comment by cemlyn007** [comment]
Yes you can, here is a small snippet where I set the colour:
```
config = brax.Config(dt=0.05, substeps=20, dynamics_mode='pbd')
torso = config.bodies.add(name='Ant Torso', mass=10)
torso.colliders.add(color='#ff0000', capsule={'radius': 0.25, 'length': 0.5})
```
Hope this helps.
